npm/npm/dependencies.json

```diff
@@ -0,0 +1,56 @@
+{
+   "[3.0, 4.0)": [
+      {
+         "tool": "node",
+         "versionRange": "[0.8, 7.0)"
+      }
+   ],
+   "[4.0, 5.0)": [
+      {
+         "tool": "node",
+         "versionRange": "[0.8, 8.0)"
+      }
+   ],
+   "[5.0, 6.0)": [
+      {
+         "tool": "node",
+         "versionRange": "[4.0,)"
+      }
+   ],
+   "[6.0, 7.0)": [
+      {
+         "tool": "node",
+         "versionRange": "[4.0,)"
+      }
+   ],
+   "[7.0, 8.0)": [
+      {
+         "tool": "node",
+         "versionRange": "[6.0,)"
+      }
+   ],
+   "[8.0, 9.0)": [
+      {
+         "tool": "node",
+         "versionRange": "[10.0, 19.0)"
+      }
+   ],
+   "[9.0, 10.0)": [
+      {
+         "tool": "node",
+         "versionRange": "[18.0.0,)"
+      }
+   ],
+   "[10.0, 11.0)": [
+      {
+         "tool": "node",
+         "versionRange": "[18.0.0,)"
+      }
+   ],
+   "[11.0,)": [
+      {
+         "tool": "node",
+         "versionRange": "[20.5.0,)"
```

Comment:
Looks like great work.
I just wonder how you actually have been able to get this information...
Do you happen to have the link to the source from where you got this?
If so please add it as a link in the issue summary as we have done for other stories about `dependencies.json`.
This will be very helpful if we later need to update such files as we do not have to do the research again to find such source.